### DISCUSSION TITLE
Account suspended without any notice

### DISCUSSION THREAD

**Discussion by rebeloperAlpar** [community/community#49079] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My coworker's account just got suspended without any warning yesterday. No email reasoning why, no warning of any wrong doing. Just got locked out in a snap. All we know is that he got suspended because of violation the Terms of Service.

This is an awful way of handling the situation from GitHub's part. Maybe he should have got a warning beforehand, maybe he should have got an automatic email explain why he got suspended in more detail. Nothing. Now he is at the mercy of the GitHub Support Team which I do not read good reviews about. Like response time can be weeks? Are we serious? Like my friends livelihood is on the line here. If he's done something wrong tell him to correct it and let's move on. Totally non-transparent way of handling this and was super surprised this low level of customer care even existed in a top notch organisation such as GitHub.

Anyway, is there a way to propagate his case? Hi ticket id is: 2038709. If anybody can help we highly appreciate it.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there 👋🏼 

These measures are in place to protect not just the user's account, but all GitHub users - however, I do welcome the feedback. Support is the only way to resolve this - the community and the moderator team are unable to assist nor escalate so we'll be locking this post. I did check on the ticket and can confirm it is in the right place so I appreciate everyone's patience. Best of luck!
